Source/RxSwift/retryWithBehavior.swift

```diff
@@ -17,6 +17,8 @@ Specifies how observable sequence will be repeated in case of an error
 Delay will be incremented by multiplier after each iteration (multiplier = 0.5 means 50% increment)
 - CustomTimerDelayed: Will be repeated specified number of times. Delay will be calculated by custom closure
 */
+
+// swiftlint:disable identifier_name
```

Comment:
If you disable rules you need to re-enable them later. What's the issue here? I'm not seeing a warning. If so, please make it part of a separate PR :) 

I prefer you pull out the rest if poissible.